### PR TITLE
postgresql: InstallDev pkg-config files for libpq

### PR DESCRIPTION
**Pull request by michael-dev**
Signed-off-by: Michael Braun <michael-dev@fami-braun.de>

Please also read https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md

Maintainer: daniel@makrotopia.org
Compile tested: OpenWRT P1020WLAN

### CODE REVIEW

File: libs/postgresql/Makefile [hunk 1]
```diff
@@ -222,6 +222,11 @@ define Build/InstallDev
 	$(CP) $(PKG_INSTALL_DIR)/usr/include/postgresql $(1)/usr/include/
 	$(INSTALL_DIR) $(1)/usr/lib
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpq.{a,so*} $(1)/usr/lib/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/pg_config $(1)/usr/bin/
```
Comment (dangowrt) [inline libs/postgresql/Makefile]: `pg_config` is an executable compiled for the target, it cannot just go into staging_dir/host as it typically cannot be executed on the host. We also already install a host-compiled version of it in `Host/Install`, so no need to overwrite it with the target-compiled version in InstallDev.

File: libs/postgresql/Makefile [hunk 2]
```diff
@@ -222,6 +222,11 @@ define Build/InstallDev
...
+	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libpq.pc $(1)/usr/lib/pkgconfig/
+	$(SED) 's,-L$$$${exec_prefix}/lib,,g' $(1)/usr/bin/pg_config
```
Comment (dangowrt) [inline libs/postgresql/Makefile]: ... so no need to do this

File: libs/postgresql/Makefile [hunk 3]
```diff
@@ -222,6 +222,11 @@ define Build/InstallDev
...
+	[ -n "$(TARGET_LDFLAGS)" ] && $(SED) 's#$(TARGET_LDFLAGS)##g' $(1)/usr/lib/pkgconfig/libpq.pc || true
```
Comment (dangowrt) [inline libs/postgresql/Makefile]: ... and this as well.

**Comment by dangowrt** [comment]
@michael-dev Are you still working on this?

**Comment by michael-dev** [comment]
I'll look into it but am currently busy.

**Comment by michael-dev** [comment]
Should be fixed. Thanks for pointing this out.

**Pull request merged**